src/storage/src/source/sql_server/replication.rs

```diff
@@ -349,7 +396,7 @@ type StackedAsyncOutputHandle<T, D> = AsyncOutputHandle<
 /// Helper method to return a "definite" error upstream.
 async fn return_definite_error(
     err: DefiniteError,
-    outputs: &[usize],
+    outputs: impl Iterator<Item = &u64>,
```

Comment:
This function probably prefers just `u64` since it unconditionally dereferences. You can add a `.copied()` in the `.iter()` calls in the flatmap above to get a `u64` from a `&u64` iterator